lib/rule-tester/rule-tester.js

```diff
@@ -50,7 +50,7 @@ const
     { getRuleOptionsSchema, validate } = require("../shared/config-validator"),
     { Linter, SourceCodeFixer, interpolate } = require("../linter");
 
-const ajv = require("../shared/ajv")({ strictDefaults: true });
+const ajv = require("../shared/ajv")({ strictSchema: true });
```

Comment:
This is a breaking change for plugins, because their tests will fail for all rules with schemas that don't pass [strictSchema validations](https://ajv.js.org/strict-mode.html#prohibit-ignored-keywords).

(enabling `strictDefaults` in `RuleTester` was also treated as a breaking change: https://github.com/eslint/eslint/issues/11473)